### PR TITLE
The heatmap of Smashing Magazine Homepage

### PR DESCRIPTION
**Pull request by smcommits**
This is the project of HTML and CSS in which we created a heatmap the home page of https://www.smashingmagazine.com/. The aim was to highlight the attractive parts in dark colors and the unattractive parts in ligth color.

Feature:
- Used dark gray for focused parts and light gray for unfocused parts
- Used typogorpahy which is imported from google fonts.
- Contains hover effects replicated from original website
- Contains animations replicated from the original website

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Amazing job so far, guys! 👍 
Your linters are green 🟢 
Clean code 💯 
Professional readme file 👏 
Very good use of float, grid and flex 🥇 
I am impressed! Keep it up, guys! 😃 

However, we need to make some changes before approving the project 💪 
Check my comments below 👇  make the changes and resubmit your project for a new code review

Good luck, team!

## Changes

- Kindly, remove the horizontal scrollbar
- There are some important heat points missing
   - 'Seriously, Ted?' on the upper right side of the main section
Capture

   - In the latest post section, you only have 1 add in the second row, must be 2.
Capture1

   - The banner after that section should not be 100% width
Capture2

  [OPTIONAL]

- It is a good practice to use semantic HTML tags, call `footer` to the footer, `main` to the main section, `section` to each of the different sections in your page. It makes your code more readable and easy to follow.
- Improve your commit messages, make them explain what you did with a bit more of detail, so if you need to roll back you know where exactly made the changes.

File: index.html
Comment (teekaytech) [inline index.html]: These footer items should be linked. For instance, we should have something like:
```
<ul>
 <li><a href=""></a></li>
 <li><a href=""></a></li>
</ul>
```
Although content is not important. Yet, this is a requirement.

File: index.html
Comment (teekaytech) [inline index.html]: These should be linked too.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__I must say, you've done an amazing job so far. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] You should link the items in your footer section. Check my comments below for details.






**[ COMMENTS ]**

- [ ] You should be careful of the way you use the `!important` rule in your CSS. If possible, try to avoid it. If not, using it in 1 or 2 places (like you did) is still okay. Overusing it can mess things up.


**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by javitocor — APPROVED** [review]
# STATUS APPROVED 🥇 

Congratulations team, you have met all the requirements and your project is approved! 👏 
Do not forget to fill up the completion form and you can move into the next project ⏩ 

Good luck with it! 😃

**Pull request merged**
